types/black-engine/index.d.ts

```diff
@@ -0,0 +1,168 @@
+// Type definitions for black-engine 0.5
+// Project: https://github.com/MassiveHeights/Black (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: David Dunnings <https://github.com/Dunnings>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.3
```

Comment:
is there a specific reason (which language feature) this is fixed to 4.3? (any restriction of this kind limits package interop with other types)

Reply:
Child classes sometimes use the "override" keyword to change the function signature of parent functions. 

For example, the [VectorCurveScatter](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/37ddde8e584ba1ebd9ceb124712eec4af27969c6/types/black-engine/scatters/VectorCurveScatter.d.ts) class overrides the getValueAt function to return a **Vector** as opposed to **any** in the base class [Scatter](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/37ddde8e584ba1ebd9ceb124712eec4af27969c6/types/black-engine/scatters/Scatter.d.ts):
 
`override getValueAt(t: number): Vector;`

What would you recommend in this case?
If Isimply remove the override keyword, it would allow for typescript < 4.3 but I'm not sure if it would throw errors complaining that the type definition is incorrect in the child class.

Reply:
If I remove the override keyword then Typescript >= 4.3 will throw an error unless `noImplicitOverride` is defined in the tsconfig compilerOptions.

Reply:
Okay, after some research I've concluded it's better to not include the override keyword in order to increase TS compatibility. I've removed all override keywords and removed the dependency on TS >= 4.3 in this commit https://github.com/DefinitelyTyped/DefinitelyTyped/pull/59390/commits/f95881d71d63214793a7d75107c1a6aae58fdcc8.

Reply:
Thanks!